datus/tools/func_tool/context_search.py

```
@@ -203,3 +233,245 @@ def search_reference_sql(
+                domain="",
+                layer1="",
+                layer2="",
+                top_n=top_n,
+            )
+            knowledge_list = result.to_pylist()
+            filtered_knowledge = [
+                {"terminology": item["terminology"], "explanation": item["explanation"]} for item in knowledge_list
+            ]
+            logger.debug(f"result of search_knowledge: {filtered_knowledge}")
+            return FuncToolResult(success=1, error=None, result=filtered_knowledge)
+        except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)